### PR TITLE
WNYC-163 White Label Whats On Snapshot

### PR DESCRIPTION
**Pull request by bkammin**
This PR implements the design changes detailed in WNYC-163 and refactors `nypr-o-main-audio-player` toward conforming to the Radial component guidelines. It also the renames the component to `organism/whats-on`.

I'm still working on:

- accessibility tests
- code documentation
- storybook stories

### CODE REVIEW

File: addon/components/organism/whats-on/schedule/template.hbs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div class='whats-on-schedule-metadata'>
+    <div class='whats-on-schedule-metadata-image'>
+        <img
+            alt="image for {{airing.showTitle}}"
+            src={{airing.show.image.url}}>
+    </div>
+    <div class="whats-on-schedule-metadata-text-outer">
```
Comment (walsh9) [inline addon/components/organism/whats-on/schedule/template.hbs:7]: why is this text-outer? where's the inner?  am i missing something?

File: app/styles/_library/_config.imports.scss [hunk 2]
```diff
@@ -127,7 +127,7 @@ $tests: false;
 @import "nypr-m-toggle-box";
 @import "nypr-o-persistent-audio-player";
 @import "nypr-m-play-button";
-@import "nypr-o-main-audio-player";
+@import "./components/whats-on";
```
Comment (walsh9) [inline app/styles/_library/_config.imports.scss:130]: When you rebase to pull in the button branch there's a new file for all these component imports, `app/styles/_library/_components.scss`

File: app/styles/_library/components/_whats-on.scss [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+      .whats-on-episode-segment {
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        padding-bottom: 10px;
+        padding-top: 10px;
+        width: 100%;
+
+        .whats-on-episode-segment-dot {
+          background-color: #EAEFF0;
```
Comment (walsh9) [inline app/styles/_library/components/_whats-on.scss:211]: It's a bit weird that the designs aren't using one of our white label colors here. Really would like to use tokens and not hard-coded colors especially in the base white label styles. I understand that's this is what is in the designs, maybe we can talk to design about this.

File: app/styles/_library/components/_whats-on.scss [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+        display: flex;
+        flex-direction: row;
+
+        .whats-on-schedule-metadata-text-outer-title-outer {
+          flex: 1;
+
+          .whats-on-schedule-metadata-text-outer-title-outer-label {
+            color: rgb(var(--color-primary-1));
+            font-family: Open Sans;
+            font-size: 11.237px;
```
Comment (walsh9) [inline app/styles/_library/components/_whats-on.scss:283]: Even worse if we can't stick to our font scale tokens. I would definitely ask design about these.

Comment (bkammin) [reply app/styles/_library/components/_whats-on.scss:283]: yep I've already got a question out to design for this

File: app/styles/_library/_config.imports.scss (hunk 2 above)
Comment (bkammin) [reply app/styles/_library/_config.imports.scss:130]: done

File: addon/components/organism/whats-on/episode-content/template.hbs [hunk 5]
```diff
@@ -0,0 +1,40 @@
+{{#if segments}}
+    <!-- todo: display segments from api when they are available -->
+    <div class="whats-on-episode-segments">
```
[the remark this answers is not in the document]
Comment (bkammin) [reply addon/components/organism/whats-on/episode-content/template.hbs:3]: switching to `<ul>` and adding link to these notes to the `todo`

File: addon/components/organism/whats-on/schedule/template.hbs (hunk 1 above)
Comment (bkammin) [reply addon/components/organism/whats-on/schedule/template.hbs]: renamed these classes (used `copy` as in its `written matter` context)

Comment (bkammin) [reply addon/components/organism/whats-on/schedule/template.hbs]: also broke the `schedule` component styles out into their own file and renamed the classes accordingly, in the same manner as `episode`.

**Pull request merged**